src/ResourceManager/Sql/Commands.Sql/ManagedDatabase/Cmdlet/AzureSqlManagedDatabaseCmdletBase.cs

```diff
@@ -0,0 +1,44 @@
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using Microsoft.Azure.Commands.Sql.Common;
+using Microsoft.Azure.Commands.Sql.ManagedDatabase.Services;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Sql.ManagedDatabase.Cmdlet
+{
+    public abstract class AzureSqlManagedDatabaseCmdletBase<TModel> : AzureSqlCmdletBase<TModel, AzureSqlManagedDatabaseAdapter>
+    {
+        /// <summary>
+        /// Gets or sets the name of the managed instance to use.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 1,
+            HelpMessage = "The name of the Azure SQL Database Managed Instance.")]
```

Comment:
The help text doesn't need to say "Azure SQL Database" over and over again - this is annoying for customer to read. The cmdlet help can just say that once in the synopsis, then the parameter help can be like "The name of the instance".